contribute/breaking-changes-guide.md

```diff
@@ -0,0 +1,120 @@
+- `@grafana/toolkit`
+- `@grafana/ui`
+- etc. ([View all available on npmjs.com](https://www.npmjs.com/org/grafana))
+
+Any change that causes dependent software to behave differently is considered to be breaking.
+
+## What is Levitate?
+
+[`@grafana/levitate`](https://github.com/grafana/levitate) is a tool created by Grafana that can show breaking changes between two versions of a **TypeScript** package or a source file.
+
+It can list exported members of an NPM package or imports used by an NPM package,
+**but we are mainly using it for comparing different versions of the same package to see changes in the exported members.**
+
+We have a Github workflow that runs against every pull request and comments a hint in case there are
+possible breaking changes. It also adds the `"breaking change"` label to the pull request.
+
+## How does the CI workflow look like?
+
+<img src="./breaking-changes-workflow.png" alt="CI workflow" width="700" />
+
+## I received a comment on my PR, what does it mean?
+
+![Levitate comment](./breaking-changes-comment-screenshot.png)
+
+Receiving a comment like the one above does not necessarily mean that you actually introduced breaking
+changes (as certain edge cases are still not covered by the tool), but as there is a good chance we rather raise attention.
+
+By clicking the links in the comment ("more info" or "Check console output") you can see the actual console
+output of the tool which gives more detailed information on what triggered the notification.
+
+**Removed exported members** (consolve view):<br />
+This means that some previously exported members won't be available in the newer version of the package which can break dependent plugins.
+
+![](./breaking-changes-console-screenshot-1.png)
+
+**Changed an existing member** (consolve view):<br />
+This means that a member was changed in a way that can break dependent plugins.
+
+![](./breaking-changes-console-screenshot-2.png)
+
+**No breaking changes** (consolve view):<br />
+Seeing this does not mean that you made no changes, but that most probably none of them were breaking. You are good to go! 👏
+
+![](./breaking-changes-console-screenshot-3.png)
+
+## How can I decide if it is really a breaking change?
+
+First go to the console output of the workflow and make sure that the diffs make sense.
+
+It can happen that Levitate highlights a change which is marked with TSDoc tags `// @alpha` or `// @internal` in
+which case you can choose to ignore it - keep in mind though that these flags won't really hold developers back
+from using your code and most likely it is going to cause them problems if we are breaking them.
+
+It can also happen that Levitate marks changing an interface as a possible breaking change.
+While it is correct (if anyone implements that interface introducing a new property will break their code),
+maybe it is an interface that is basically never implemented by other developers, in which case you
+can choose to ignore Levitate's message.
```

Comment:
```suggestion
It can also happen that Levitate marks changing an interface as a possible breaking change. 
For anyone that implements that interface introducing a new property will break their code. Whilst this is correctly marked as a breaking change maybe it is an interface that is never implemented by other developers. In which case you can choose to ignore Levitate's message.
```